test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -455,9 +455,8 @@ def test_successful_authorize_with_3ds
     )
     assert first_message = @gateway.authorize(@amount, @threeDS_card, options)
     assert first_message.test?
+    assert first_message.success?
     refute first_message.authorization.blank?
-    refute first_message.params['issuer_url'].blank?
```

Comment:
worldpay does not seem to send these two parameters back in their responses, nor can I find reference to them in their docs (though I'm sure they used to send them back?) That being said the purpose of the test is to prove successful transactions which I believe adding the `.success?` line above covers.